src/components/CreditCard/index.tsx

```diff
@@ -0,0 +1,64 @@
+import {
+  CardContainer,
+  CardContainerFront,
+  CardContainerHeader,
+  CardContainerContent,
+  CardContainerContentText,
+  CardContainerFooter,
+  CardContainerFooterSpan,
+  CardContainerHeaderitems,
+  CardContainerHeaderitemsFlipped,
+  CardContainerHeaderBlackLine,
+  CardContainerContentBackCard,
+} from './style'
+import { CreditCardProps } from './type'
+
+const CreditCard = ({
+  name = 'Nome Completo',
+  nameColor = '#707070',
```

Comment:
Consider "nameColor" and "cardNumberColor" to be just one property.

Reply:
Done, check it out here - f437f60576bfae98b9de3e3a80a6bdcaff7fdeb4